.github/workflows/deploy-prod.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy DotNet project to Azure Function App
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+env:
+  AZURE_FUNCTIONAPP_NAME: "Prod-STORIS-URL-Redirect"
+  AZURE_FUNCTIONAPP_PACKAGE_PATH: "."
+  DOTNET_VERSION: "6.0.x"
+
+jobs:
+  build-and-deploy:
+    runs-on: windows-latest
+    environment: prod
+    steps:
+      - name: "Checkout GitHub Action"
+        uses: actions/checkout@v3
+
+      - name: "Login via Azure CLI"
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_RBAC_CREDENTIALS_PROD }}
+
+      - name: Setup DotNet ${{ env.DOTNET_VERSION }} Environment
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+
+      - name: "Resolve Project Dependencies Using Dotnet"
+        shell: pwsh
+        run: |
```

Comment:
Would this work without using powershell to change the working directory?

Reply:
`dotnet build` states you either need to specify the project/solution file in the command or run it from the directory that the file is located in. I suppose for this repo that's not needed with the files in the root, but it would be necessary for TinyBlazorAdmin.

Even if it's not necessary, I feel it could make sense to leave as is just for consistency with the other repo or if we decided to move the files.

Reply:
I'm good with that. I thought dotnet build does let you It does allow you to use wildcards like `**/*.csproj`, but that might be the Azure Pipelines DotNetCoreCLI@2 task that I'm remembering that has a projects attribute. https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/dotnet-core-cli-v2?view=azure-pipelines